tox.ini

```diff
@@ -8,13 +8,6 @@ envlist =
        {py27,py34,py35}-django{19}
 
 [testenv]
-basepython =
-            py27: python2.7
-            py32: python3.2
-            py33: python3.3
-            py34: python3.4
-            py35: python3.5
-
```

Comment:
@ryanhiebert that does seem valid, but we suggest perhaps submitting that as a separate PR. That way we can discuss and progress both separately.